sig-security/joint-review.md

```diff
@@ -0,0 +1,375 @@
+* **CNCF Requests.** *In the initial draft, please include whatever you believe the CNCF could assist with that would increase security of the ecosystem.*
+  - Assess KubeEdge CI workflows and provide security hardening methods.
+  - According to the threat model, provide a way to harden security from the overall perspective.
+
+## Appendix
+
+### Known Issues Over Time.
+
+For details, please see: 
+
+https://github.com/kubeedge/kubeedge/security/advisories
+
+### Case Studies
+
+KubeEdge has been widely adopted in industries including transportation, energy, Internet, CDN, manufacturing, smart campus etc.
+
+* Xinghai IoT is an IoT company that provides comprehensive smart building solutions by leveraging a construction IoT platform, intelligent hardware, and AI. Xinghai IoT built a smart campus with cloud-edge-device synergy based on KubeEdge and its own Xinghai IoT cloud platform, greatly improving the efficiency of campus management. With AI assistance, nearly 30% of the repetitive work is automated. In the future, Xinghai IoT will continue to collaborate with KubeEdge to launch KubeEdge-based smart campus solutions.
+* In China’s highway electronic toll collection (ETC) system, KubeEdge helps manage nearly 100,000 edge nodes and more than 500,000 edge applications in 29 of China’s 34 provinces, cities and autonomous regions. With these applications, the system processes more than 300 million data records daily and has improved traffic efficiency at the toll stations by 10 times.
+* China Mobile On-line Marketing Service Center, is a secondary organ of the China Mobile Communications Group, holds the world’s largest call center with 44,000 agents, 900 million users, and 53 customer service centers, builds a cloud-edge synergy architecture consisting of two centers and multiple edges based on KubeEdge. See more details [here](https://www.cncf.io/blog/2021/08/16/china-mobile-kubeedge-based-customer-service-platform-featuring-edge-cloud-synergy).
+* e-Cloud of China Telecom, uses KubeEdge to manage CDN edge nodes, automatically deploy and upgrade CDN edge services, and implement edge service disaster recovery (DR) when it migrates its CDN services to the cloud. See more details [here](https://www.cncf.io/blog/2022/03/18/e-cloud-large-scale-cdn-using-kubeedge).
+
+### Related Projects / Vendors
+
+Another edge computing that relative to Kubernetes is OpenYurt, it uses a centralized Kubernetes control plane residing in the cloud site to manage multiple edge nodes residing in the edge sites. OpenYurt manage edge nodes through native Kubelet component, each edge node has moderate compute resources available in order to run edge applications plus the required OpenYurt components. It provides a edge node daemon YurtHub that serves as a proxy for the outbound traffic from typical Kubernetes node daemons such as Kubelet, Kubeproxy, CNI plugins, etc.

```

Comment:
This part we can describe more about related projects those are working on edge computing scenarios and can be leveraged by KubeEdge to enforce better user/developer experience when building cloud-edge platforms or systems.